src/__tests__/index.js

```diff
@@ -257,17 +257,20 @@ const shortTests = [
   [[{transform: 'translateX( 30px )'}], {transform: 'translateX( -30px )'}],
   [[{transform: 'translateX(30%)'}], {transform: 'translateX(-30%)'}],
   [
-    [{transform: 'translateY(30px) rotate(20deg) translateX(10px)'}],
-    {transform: 'translateY(30px) rotate(20deg) translateX(-10px)'},
+    [{transform: 'translateY(30px) scale(2) translateX(10px)'}],
+    {transform: 'translateY(30px) scale(2) translateX(-10px)'},
   ],
   [
-    [{transform: 'translateX(30px) rotate(20deg) translateY(10px)'}],
-    {transform: 'translateX(-30px) rotate(20deg) translateY(10px)'},
+    [{transform: 'translateX(30px) scale(2) translateY(10px)'}],
+    {transform: 'translateX(-30px) scale(2) translateY(10px)'},
   ],
   [
     [{transform: 'translate3d(30%, 20%, 10%)'}],
     {transform: 'translate3d(-30%, 20%, 10%)'},
   ],
+  [[{transform: 'rotate(100deg)'}], {transform: 'rotate(-100deg)'}],
+  [[{transform: 'rotateZ(100deg)'}], {transform: 'rotateZ(-100deg)'}],
+  [[{transform: 'rotateY(100deg)'}], {transform: 'rotateY(-100deg)'}],
```

Comment:
Yeah, I'm pretty sure we only want to flip left and right axes and want to avoid flipping other axes.

@garrettberg, if you need to flip the others I think you'll have to do that in your app code yourself. I think this would have adverse impacts on other projects.

Reply:
`rotateX` is rotation over the X axis (e.g., towards or away from you), so it's unrelated by LTR/RTL.

There is a `rotate3d` function that I've *never* seen used, but that should only be partially flipped. The regex for that would be incredibly complex though so I left it out of this PR. For reference though:

```css
.ltr {
  transform: rotate3d(1, 1, 1, 100deg);
}

.rtl {
  transform: rotate3d(1, -1, -1, 100deg);
}
```

Reply:
Yeah, we can deal with `rotate3d` later if we want :+1:

As for `rotateX`, I think it's good to keep it as you have it. Don't want to change that one. For `rotateZ`, I'm honestly not certain what to do about this. My app at PayPal doesn't use it (we do use `rotateY`). So I'm easily swayed either way on that. For `rotateY` I think that makes sense to have it converted. So yeah, just `rotateZ` I'm not sure...

Reply:
`rotateZ` is just the 3d transform version of `rotate` — they're the same visually.

Reply:
I think I'm good with it :+1:

Let's wait to hear back from @TxHawks :+1: